src/GoogleSignin.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+
+import {
+  View,
+  NativeAppEventEmitter,
+  NativeModules,
+  requireNativeComponent,
+  ViewPropTypes,
+  Platform,
+} from 'react-native';
+
+const { RNGoogleSignin } = NativeModules;
+
+class GoogleSignin {
+  // TODO vonovak kill state in this module
+  _user = null;
+  signinIsInProcess = false;
+
+  hasPlayServices(params = { autoResolve: true }) {
+    if (Platform.OS === 'ios') {
```

Comment:
maybe we can create a constant called isIOS so we don't need to rewrite this all the time...

Reply:
yes, will do